README.md

```diff
@@ -15,33 +15,33 @@ When persisting this collection of values (Mysql, Elasticsearch), we use intensi
 
 ## Problems
 
-Do not that in theory, these problems should be highlited by the use of the profiler.
+Do note that, in theory these problems should be highlited by the use of the profiler.
 It's the role of the profile to determine if an optimization worths an improvement or not.
 
-Here, we are doing the opposite: it is to better understand the problem in the Blackfire profile.
+Here, we are doing the opposite: it is better to understand the problem in a Blackfire profile.
```

Comment:
no, "it is to better understand" is the real signification

Reply:
Quite weird, we have to find a way to reformulate it.

Reply:
https://www.linguee.fr/anglais-francais/traduction/to+better+understand.html

Reply:
Ok :D